analyses/chromosomal-instability/util/chr-break-calculate.R

```diff
@@ -0,0 +1,233 @@
+# Functions for chromosomal instability calculations
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2020
+
+make_granges <- function(break_df = NULL,
```

Comment:
Is this intended to be, effectively, a private function? As in - do you foresee it being used outside of `break_density()`? Thinking about sanity checks where you say to check your `col` arguments, whether this will make sense to users who are calling `break_density()` directly. I think what you have is likely _fine_...

Reply:
It will be used by another function I need to make for another part of this analysis besides `break_density` and it *could* be used in other scenarios, which is why it isn't all one function. Is there something you recommend I do to make the error message more easily interpretable whether in or outside another function?

Reply:
As long as you will be able to interpret any potential error messages when called from another function you don't need to change anything. But, for example, you'd want to make sure that future functions that call this also have `cols` type arguments.